config/site/Rakefile

```diff
@@ -151,6 +151,30 @@ module ElasticGraph
           ::File.write(searchable_content_file, ::YAML.dump(searchable_content))
         end
 
+        desc "Extracts the site's documenation into one LLM-friendly file"
+        task extract_llm_txt_content: [:build_docs] do
+          require_relative "support/llm_content_extractor"
+
+          # Build Jekyll content so that we have parsable HTML to extract content from.
+          # (We couldn't figure out a way to extract the rendered content within Jekyll itself).
+          run_jekyll :build
+
+          output_file = SITE_SOURCE_DIR / "llms-full.txt"
+
+          extractor = ElasticGraph::LLMContentExtractor.new(
+            jekyll_site_dir: JEKYLL_SITE_DIR,
+            docs_dir: DOCS_DIR,
+            output_file: output_file
+          )
+
+          extractor.extract_and_write_content
```

Comment:
It's odd/inconsistent that in `extract_searchable_content`, the extractor just returns a string, and the rake task writes to disk, whereas here the extractor writes to disk.  If we're going to keep them as separate tasks with separate extractors it'd be nice to make them consistent.  OTOH if we merge them as I suggest in another comment then either approach is fine (since it'll just be one extractor). And it's probably simpler to have the extractor write to disk since it would write multiple files.